spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -317,10 +318,13 @@ protected void validateProperty(Snapshot snapshot, String property, Set<String>
   }
 
   protected void validateProperty(Snapshot snapshot, String property, String expectedValue) {
-    String actual = snapshot.summary().get(property);
-    assertThat(actual)
-        .as("Snapshot property " + property + " has unexpected value.")
-        .isEqualTo(expectedValue);
+    if (null == expectedValue) {
```

Comment:
these changes will show the content of the snapshot summary in case the assertion fails